test/interpreter/function-vara.spec.ts

```diff
@@ -0,0 +1,24 @@
+import {ErrorType, HyperFormula} from '../../src'
+import {adr, detailedError} from '../testUtils'
+import {ErrorMessage} from '../../src/error-message'
+
+describe('Function VARA', () => {
+  it('should take at least two arguments', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=VARA()'],
+      ['=VARA(1)']
+    ])
+
+    expect(engine.getCellValue(adr('A1'))).toEqual(detailedError(ErrorType.NA, ErrorMessage.WrongArgNumber))
+    expect(engine.getCellValue(adr('A2'))).toEqual(detailedError(ErrorType.DIV_BY_ZERO))
+  })
+
+  it('should calculate variance (sample)', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=VARA(2, 3)'],
+      ['=VARA(B2:D2, E2, F2)', 2, 3, 'foo', 'bar', 4],
+    ])
+    expect(engine.getCellValue(adr('A1'))).toEqual(0.5)
+    expect(engine.getCellValue(adr('A2'))).toEqual(3.2)
+  })
+})
```

Comment:
Missing tests:

- Arguments can be the following: numbers; names, arrays, or references that contain numbers; text representations of numbers; or logical values, such as TRUE and FALSE, in a reference.
- Logical values and text representations of numbers that you type directly into the list of arguments are counted.
- Arguments that contain TRUE evaluate as 1; arguments that contain text or FALSE evaluate as 0 (zero).
- If an argument is an array or reference, only values in that array or reference are used. Empty cells and text values in the array or reference are ignored.
- Arguments that are error values or text that cannot be translated into numbers cause errors.
- If you do not want to include logical values and text representations of numbers in a reference as part of the calculation, use the VAR function.


Source:
https://docs.oasis-open.org/office/OpenDocument/v1.3/csprd02/part4-formula/OpenDocument-v1.3-csprd02-part4-formula.html#__RefHeading__1018788_715980110
https://support.microsoft.com/en-us/office/vara-function-3de77469-fa3a-47b4-85fd-81758a1e1d07